command/commands.go

```diff
@@ -484,6 +484,11 @@ func Commands(metaPtr *Meta, agentUi cli.Ui) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
+		"operator debug": func() (cli.Command, error) {
```

Comment:
Maybe a comment that this is an alias that left for compatibility reasons but without us documenting it anymore?

Reply:
done